test/cppsuite/configs/operations_test_stress.txt

```diff
@@ -50,6 +50,12 @@ workload_generator=
         ops_per_transaction=(max=100,min=50),
         thread_count=20
     ),
+    remove_config=
+    (
+        op_rate=100ms,
+        ops_per_transaction=(max=20,min=10),
```

Comment:
Lets make min, max, 50, 100 so its a bit more stressful.

Reply:
Done in 2b4671a321ed4c502730d86ee40d72e849388761